api/src/utils/xlsx-utils/worksheet-utils.ts

```diff
@@ -149,6 +149,11 @@ export const getWorksheetRowObjects = (worksheet: xlsx.WorkSheet): Record<symbol
     let rowHasValues = false;
 
     for (let j = 0; j <= originalRange.e.c; j++) {
+      // Skip empty headers
+      if (!headers[j]) {
+        continue;
+      }
+
       // Always add the header (key) to the row object
       rowObject[headers[j]] = undefined;
```

Comment:
Just checking: 

Does this skip empty headers check contradict the comment below about always adding the header key?

(whether its best to omit empty headers or not, I can't say)

Reply:
Yea they do contradict a bit, nice catch

Updated